### PR TITLE
LPS-196180 KB DatePicker overflows the modal

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
**Before:** 

Screenshot 2023-09-14 at 13 53 38

**Now:** 

Screenshot 2023-09-20 at 10 43 19

The problem was that `openModal()` renders the modal as an iframe, so the content could never overflow the Iframe. As suggested by PEDS team, and also for accessibility rules, it is better to render directly a `ClayModal` and avoid the Iframe. 

I would like to add some unit tests, but in the meantime this pull could be reviewed.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp [hunk 1]
```diff
@@ -359,21 +359,26 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
...
+/>
+
+<div>
+	<react:component
+		module="admin/js/components/ScheduleKBArticle"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"displayDate", editKBArticleDisplayContext.getDatePickerFormattedDisplayDate()
+			).put(
+				"isScheduled", editKBArticleDisplayContext.isScheduled()
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp:380]: Boolean variable names should not have the `is*` prefix. It should be simply "scheduled". The `is*` prefix is only used for methods.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp]: Renamed! Thanks @adolfopa

**Review by adolfopa — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
I will send the tests in a different pull. Marking this as ready for review.

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
@liferay-lima QA please, could you please check if the failures are related to this pull?

**Comment by austinchiang** [comment]
Test failures unrelated

**Comment by austinchiang** [comment]
ci:forward:force

**Pull request closed without merging**